tests/framework/results/test_interface.py

```diff
@@ -16,6 +17,106 @@ def _silly_aggregator(_: pd.DataFrame) -> float:
     return 1.0
 
 
+####################################
+# Test stratification registration #
+####################################
+
+
+def test_register_stratification(mocker):
+    def _silly_mapper():
+        return "foo"
```

Comment:
Shouldn't the mapper return values in `{"some-category", "some-other-category", "some-unwanted-category"}`? I know we're not testing that in this test, but it is a bit confusing.